README.md

```diff
@@ -29,7 +29,7 @@ It correctly bundles React in production mode and optimizes the build for the be
 
 
 
-### `docker build -f ./scripts/NodeAdmin.Dockerfile .`
+### `docker build -t europe-west3-docker.pkg.dev/hoprassociation/docker-images/hopr-admin:latest .`
```

Comment:
I do not think that this is correct. Do you really build an image if you download a ready image?